plugins/modules/zos_started_task.py

```diff
@@ -0,0 +1,1405 @@
+    required: false
+    type: str
+    aliases:
+        - identifier
+  job_account:
+    description:
+        - Option job_account specifies accounting data in the JCL JOB statement for the started
+          task. If the source JCL was a job and has already accounting data, the value that is
+          specified on this parameter overrides the accounting data in the source JCL.
+        - Only applicable when state=started otherwise ignored.
+    required: false
+    type: str
+  job_name:
+    description:
+        - When state=started job_name is a name which should be assigned to a started task
+          while starting it. If job_name is not specified, then member_name is used as job_name.
+          Otherwise, job_name is the started task job name used to find and apply the state
+          selected.
+    required: false
+    type: str
+    aliases:
+        - job
+        - task
+        - task_name
+  keyword_parameters:
+    description:
+        - Any appropriate keyword parameter that you specify to override the corresponding
+          parameter in the cataloged procedure. The maximum length of each keyword=option is 66
+          characters. No individual value within this field can be longer than 44 characters in length.
+        - Only applicable when state=started otherwise ignored.
+    required: false
+    type: dict
+  member_name:
+    description:
+        - Option member_name is a 1 - 8 character name of a member of a partitioned data set that
+          contains the source JCL for the task to be started. The member can be either a job or a
+          cataloged procedure.
+        - Only applicable when state=started otherwise ignored.
+    required: false
+    type: str
+    aliases:
+        - member
+  parameters:
+    description:
+        - Program parameters passed to the started program, which might be a list in parentheses or
+          a string in single quotation marks
+    required: false
+    type: list
+    elements: str
+  retry:
+    description:
+        - I(retry) is applicable for only FORCE TCB.
+        - Only applicable when state=forced otherwise ignored.
+    required: false
+    type: str
+    choices:
+        - 'YES'
+        - 'NO'
+  reus_asid:
+    description:
+        - When REUSASID=YES is specified on the START command and REUSASID(YES) is specified in the DIAGxx parmlib member,
+          a reusable ASID is assigned to the address space created by the START command. If REUSASID=YES is not specified
+          on the START command or REUSASID(NO) is specified in DIAGxx, an ordinary ASID is assigned.
+        - Only applicable when state=started otherwise ignored.
+    required: false
+    type: str
+    choices:
+        - 'YES'
+        - 'NO'
+  state:
+    description:
+        - The desired state the started task should be after the module is executed.
+        - If state=started and the started task is not found on the managed node, no action is taken,
```

Comment:
I think this behavior should be changed, just checked the ansible.services module and when a service is not found in the managed node it fails
```
fatal: [zvm]: FAILED! => {"ansible_facts": {"discovered_interpreter_python": "/usr/bin/python3.12"}, "changed": false, "msg": "Could not find the requested service hpptt: host"}
```

This seems more aligned to the idea of idempotency, returning a succesful response with changed: false would give a false sense that the task is running already, even though we can start more than one.